### ISSUE TITLE
Get the current domain and show it in the fwd

### ISSUE THREAD

**Issue by jtagcat** [prologic/pastebin#2]
![image](https://user-images.githubusercontent.com/38327267/62004344-46b0d600-b113-11e9-9aee-947ca6087a5f.png)
should be https://url.mills.io/4IEpz instead

**Comment by taigrr** [comment]
One last one to close!

**Comment by jtagcat** [comment]
It's fun to see how I have no memory whatsoever of the issues in the prologic projects I know I once came in to do.

**Comment by prologic** [comment]
One day I'll build a full office suite of all these apps 🤣

**Comment by taigrr** [comment]
Yeah I really like this one in particular, but I am having a lot of issues with it myself. I forked it and I'll try to fix them once I figure out why they're happening. I'll submit more PRs at that point too. But I wanted to help clean up this one first :)

**Comment by taigrr** [comment]
Could we get this closed @prologic

**Comment by prologic** [comment]
> Could we get this closed @prologic

Is it fixed? I didn't see a PR?

**Comment by prologic** [comment]
I think this is for [prologic/shorturl](https://github.com/prologic/shorturl) no?
